docs/pages/oauth.rst

```diff
@@ -230,44 +230,52 @@ pass a string containing each scope that you want (if you want several, they sho
++=================+====================================================================================================================================+==========================================================================================================================================================================================================+
+| creddits        | Spend my reddit gold creddits on giving gold to other users.                                                                       | gild                                                                                                                                                                                                     |
++-----------------+------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| edit            | Edit and delete my comments and submissions.                                                                                       | edit, delete                                                                                                                                                                                             |
++-----------------+------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| flair           | Select my subreddit flair. Change link flair on my submissions.                                                                    | get_flair_choices, select_flair                                                                                                                                                                          |
++-----------------+------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| history         | Access my voting history and comments or submissions I've saved or hidden.                                                         | get_hidden, get_saved, get_liked, get_disliked (the last two do not require the ``history`` scope if upvoted and downvoted posts are made public via the preferences)                                    |
```

Comment:
get_liked and get_disliked should be get_upvoted and get_downvoted, as those are the official names now. Liked/Disliked will probably be removed at some point so it's good if we can make the switch.